devops/Makefile

```diff
@@ -286,7 +286,9 @@ deploy-redis:
 # Command to initialize the redis cluster.
 # Make sure that all the redis pods are up and running before initializing the cluster.
 init-redis-cluster:
-	@oc exec -n $(NAMESPACE) -it redis-0 -- redis-cli -a $(oc get secret -n $(NAMESPACE) redis-creds -o jsonpath='{.data.password}' | base64 -d) --cluster create --cluster-replicas 1 $(oc get pods -n $(NAMESPACE) -l app=redis -o jsonpath='{range.items[*]}{.status.podIP}:6379 ')
+	REDIS_PASSWORD=$$(oc get secret -n $(NAMESPACE) redis-creds -o jsonpath='{.data.password}' | base64 -d); \
+	REDIS_NODES=$$(oc get pods -n $(NAMESPACE) -l app=redis -o jsonpath='{range .items[*]}{.status.podIP}:6379 ' | sed 's/ :6379 $$/ /'); \
```

Comment:
Executing the command 
`oc get pods -n 0c27fb-dev -l app=redis -o jsonpath='{range .items[*]}{.status.podIP}:6379 ' | sed 's/ :6379 $$/ /'`
Is still returning the extra `:6379` at the end, as shown below. Is it the expectation?
`10.97.50.200:6379 10.97.38.75:6379 10.97.70.236:6379 10.97.94.99:6379 10.97.56.98:6379 10.97.101.99:6379 :6379`

Reply:
The idea is when the command is run via a makefile it trims the last 6379
<img width="1243" alt="image" src="https://github.com/bcgov/SIMS/assets/62901416/b385098e-6bea-429f-b930-1e8fb8c32486">

<img width="1045" alt="image" src="https://github.com/bcgov/SIMS/assets/62901416/c4a917c8-8995-4b72-9c85-c6f83291b4e1">

Dont ask me how it does, but it does 😄

Reply:
I see that `echo` is correctly printing it. Thanks for the explanation 😉